civis/resources/__init__.py

```diff
@@ -1,3 +1,3 @@
-from ._resources import generate_classes
+from ._resources import generate_classes, get_swagger_spec
 
-__all__ = ["generate_classes"]
+__all__ = ["generate_classes", "get_swagger_spec"]
```

Comment:
You should also add "generate_classes_maybe_cached" so that you don't need to import it from a private module.